docs/matlab.rst

```diff
@@ -0,0 +1,6 @@
+matlab
+======
+
+.. mat:automodule:: matlab
+
+.. mat:autofunction:: ecat2nii
```

Comment:
when mixing matlab and python in a project, seems that one must preface the sphynx directives (is that what those are called by `mat:`)

https://github.com/sphinx-contrib/matlabdomain#documenting-python-and-matlab-sources-together